src/modules/auth/HOC/firewall.jsx

```diff
@@ -1,5 +1,5 @@
 import { React, useSelector, Auth } from '../dependencies';
-
+import Loading from '../components/Loading';
```

Comment:
This component should be part of UI module

Reply:
Working on it in #10

Reply:
cool